Internship2022TeamPatri/Internship2022TeamPatri/Controllers/Auth/Register/RegisterViewController.swift

```diff
@@ -0,0 +1,136 @@
+//
+//  RegisterViewController.swift
+//  Internship2022TeamPatri
+//
+//  Created by Coralia Diana Muresan on 28.07.2022.
+//
+
+import UIKit
+
+class RegisterViewController: UIViewController {
+    
+    @IBOutlet weak var registerButton: UIButton!
+    @IBOutlet var backgroundView: UIView!
+    @IBOutlet weak var titleLabel: UILabel!
+    @IBOutlet weak var emailLabel: UILabel!
+    @IBOutlet weak var personalIDLabel: UILabel!
+    @IBOutlet weak var studentIDLabel: UILabel!
+    @IBOutlet weak var passwordLabel: UILabel!
+    @IBOutlet weak var view1: UIView!
+    @IBOutlet weak var view2: UIView!
+    @IBOutlet weak var view3: UIView!
+    @IBOutlet weak var view4: UIView!
+    @IBOutlet weak var emailText: UITextField!
+    @IBOutlet weak var personalIDText: UITextField!
+    @IBOutlet weak var studentIDText: UITextField!
+    @IBOutlet weak var passwordText: UITextField!
+    
+    let colorText = UIColor(red: 37/255, green: 22/255, blue: 5/255, alpha: 1)
```

Comment:
Implement UIColor extension to not normalise the colors every time. Talk with Vlad, he has the same problem.